### PR TITLE
1146 change name of user table to sidewalk_user

### LINKED ISSUE #1146: Database table 'user' conflicts with PostgreSQL function
The sidewalk database has a table named `user`. As discussed in this [StackOverflow post](https://dba.stackexchange.com/a/75559),  `user` is a function keyword in PostgreSQL.

```sql
-- returns current database user
SELECT * FROM user;

-- returns the 'user' table
SELECT * FROM "user";
```

We may want to consider renaming this table since its current name leads to confusing behavior.

### PR DESCRIPTION
**Pull request by misaugstad**
Resolves #1146 

Changes the name of the `user` table to `sidewalk_user` so that we don't have to put the name of the table in quotes or use "sidewalk.user" each time we reference the table. We no longer have a table with the same name as a postgres keyword :roll_eyes: 

NOTE: Do not review this PR until PR #1319 (updating mission infrastructure), #1327, and #1328 have been merged. I branched this code off of the code in that PR instead of the code on develop (b/c each of these PRs creates a new evolutions file, and it would be annoying to have a merge commit where I change the name of the evolutions file every time I merge a PR.

Testing:
* Test that the evolution file works correctly in that the table name changes to `sidewalk_user` when compiling and running the code on this branch, and changes back to `user` when compiling and running code on `develop` again.
* Do a few things on the website that would definitely make use of the `sidewalk_user` table (log in, audit a bit, check the user dashboard, log out) and just make sure that no errors occur.
* Search through the code to see if you can find any places where I missed a reference to the "user" table that still needs to be changed.

### CODE REVIEW

**Comment by misaugstad** [comment]
okay @manaswis this should be good to test now!

File: conf/evolutions/default/19.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# --- !Ups
+ALTER TABLE "user" RENAME TO sidewalk_user;
```
Comment (manaswis) [inline conf/evolutions/default/19.sql]: Would this work since user is a table (outside of sidewalk) in itself? Shouldn't this be sidewalk.user?

Comment (misaugstad) [reply conf/evolutions/default/19.sql]: Both `sidewalk.user` and quoted `"user"` work!

**Comment by manaswis** [comment]
Looks good!

**Pull request merged**